doc/programming/samples.rst

```diff
@@ -156,15 +156,14 @@ command-line arguments. For example, a 5 second animation at 60 frames per secon
 
 .. code-block:: Shell
 
-     render humanoid.xml 5 60 rgb.out
+     record humanoid.xml 5 60 rgb.out
 
 The default humanoid.xml model specifies offscreen rendering with 800x800 resolution. With this information in hand, we
```

Comment:
1. 800x800 is [not true](https://github.com/google-deepmind/mujoco/blob/9ac048e262e664b5951678ae5c098d9175862218/model/humanoid/humanoid.xml#L22).
2. The doc should be explicit about the importance `visual/global/{offwidth,offheight}` to be large enough or the command will fail.

Reply:
1. Thanks, fixed!
2. I added a little comment. I cannot reproduce "large enough". Works for me even for 4x4 given that the model_size and the offscreen render sizes are identical. Is there anything that I am missing? :)

Reply:
Really? You can record videos that are larger than the height/width spec ??
That's strange. I will look again tomorrow.

Reply:
Not larger nor smaller, but exactly according to the spec - I changed the offwidth and offheight in humanoid.xml to values between 4 and 4000 (same values for ffmpeg) and then could record video with such resolution.

Thank you @yuvaltassa for all your thorough feedback